### DISCUSSION TITLE
Is aero-structural optimizaiton for axial compressor blade in Transonic condition possible?

### DISCUSSION THREAD

**Discussion by Ryupuriwatt** [mdolab/dafoam#666] [General]
![image](https://github.com/user-attachments/assets/1baff40f-3427-4034-a565-fafbc9997dc8)
Profressor I have question that in transonic condition of rotor blade I can do aerodynamic optimization(torque reduce around 4%) but the problem is aero-structural optimizaiton for transonic condition the objective or torque doesn't decrease as shown in the figure so could profressor give me a suggustion. I have read DAFoam documentation and the latest tutorial there is nothing related to aerostructural optmization for transonic condition. The calculation does converge for linear and non linear solver
![image](https://github.com/user-attachments/assets/32687e16-1f00-4bba-8342-b9077318e0a9)

**Comment by friedenhe** [comment]
From the log file you showed, you had a lot of mesh deformation failures. One approach to fixing it is to reduce the bounds of your design variables, or to use mesh quality constraints.

**Comment by NeuroNexusAI** [comment]
"Absolutely! 🚀 Aero-structural optimization for axial compressor blades in transonic conditions is definitely feasible. It involves advanced computational methods to enhance performance while managing structural integrity. Exciting developments in this area! 🛠️✨"
